internal/error.go

```diff
@@ -137,6 +140,7 @@ type (
 		//
 		// NOTE: This option is supported by Temporal Server >= v1.24.2 older version will ignore this value.
 		NextRetryDelay time.Duration
+		Category       ApplicationErrorCategory
```

Comment:
What is the release status of https://github.com/temporalio/features/issues/614?

Reply:
That's a good question - I'll ask drew